pom.xml

```diff
@@ -50,6 +50,7 @@
         <!-- tomcat version &gt; 7.0.56 causes SSL to fail -->
         <tomcat.version>7.0.56</tomcat.version>
         <jackson.version>2.5.3</jackson.version>
+        <jackson.datatype.guava.version>2.4.0</jackson.datatype.guava.version>
```

Comment:
I don't think this is used can you please remove it, also in general variables are only used where the value is used multiple times, otherwise it can make it harder to read and change the versions of dependencies.